src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeModal.js

```diff
@@ -667,6 +667,7 @@ export const GcpComputeModalBase = ({
       ..._.mapKeys((key) => `desiredRuntime_${key}`, desiredRuntime),
       desiredRuntime_exists: !!desiredRuntime,
       desiredRuntime_cpus: desiredRuntime ? desiredRuntimeCpus : undefined,
+      desiredRuntime_gpuEnabled: desiredRuntime ? desiredRuntimeHasGpu : undefined,
```

Comment:
Shouldn't this be false not `undefined` in the else case?

Reply:
I thought about that, but it seems clearer to have this as `undefined` --> when the desiredRuntime doesn't exist then there is no value for desiredRuntimeHasGpu because there's no desiredRuntime to check the GPU state of, if that makes sense? I also think in the createCloudEnvironment case, which is what the goal of this is to track, there shouldn't be a case where the desiredRuntime doesn't exist.